test/integration/api/forms/list.js

```diff
@@ -45,6 +45,9 @@ describe('api: /projects/:id/forms (listing forms)', () => {
               body.forEach((form) => form.should.be.an.ExtendedForm());
```

Comment:
How about updating the `ExtendedForm()` assertion to incorporate the new `reviewState` property?